pipelines/Jenkinsfile

```diff
@@ -0,0 +1,256 @@
+    BBDD_SID        = "etendo"
+    BBDD_PORT       = "5432"
+    BBDD_SYSTEMUSER = "postgres"
+    BBDD_SYSTEMPASS = "syspass"
+    BBDD_USER       = "tad"
+    BBDD_PASSWORD   = "tad"
+    NEXUS_USER      = credentials('nexus-admin-user')
+    NEXUS_PASSWORD  = credentials('nexus-admin-passwd')
+    GITHUB_USER     = "etendobot"
+    GITHUB_TOKEN    = credentials('github-read-package-token')
+    ACCESS_TOKEN    = credentials('access_token_github')
+    EMAIL_ADDRESS   = credentials('email_builds')
+    CONTEXT_BUILD   = "Module Tests"
+
+    ETENDO_BASE_URL   = "https://github.com/etendosoftware/etendo_base"
+    DOCKER_MODULE_URL = "https://github.com/etendosoftware/com.etendoerp.docker.git"
+
+    JAVA_HOME       = "/usr/lib/jvm/java-11-openjdk-amd64"
+    MODULE_PACKAGE  = "com.etendoerp.docker"
+
+    COMMIT_INPROGRESS_STATUS = "pending"
+    COMMIT_SUCCESS_STATUS    = "success"
+    COMMIT_FAILED_STATUS     = "failure"
+
+    SUCCESS  = "SUCCESS"
+    FAILED   = "FAILED"
+    UNSTABLE = "UNSTABLE"
+  }
+
+  agent {
+    kubernetes {
+        inheritFrom 'jenkins-node-pg'
+        defaultContainer 'jnlp'
+        yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  name: jenkins-node-pg-0
+  namespace: jenkins
+  labels:
+    app.kubernetes.io/name: jenkins-node-pg
+spec:
+  volumes:
+    - name: rsa-keys
+      configMap:
+        name: rsa-keys
+        defaultMode: 384
+    - name: docker-sock
+      hostPath:
+        path: /var/run/docker.sock
+        type: ''
+  containers:
+    - name: compiler
+      image: etendo/compiler_jenkins:1.0.6
+      ports:
+        - name: ssh
+          containerPort: 22
+          protocol: TCP
+        - name: visualvm
+          containerPort: 8000
+          protocol: TCP
+      resources:
+        requests:
+          cpu: 1072m
+          memory: 4000Mi
+        limits:
+          cpu: 2072m
+          memory: 6500Mi
+      volumeMounts:
+        - name: rsa-keys
+          mountPath: /root/.ssh/
+        - name: docker-sock
+          mountPath: /var/run/docker.sock
+      lifecycle:
+        postStart:
+          exec:
+            command:
+              - bash
+              - '-c'
+              - >-
+                chmod a+x /var/run/docker.sock && rm
+                /etc/apt/sources.list.d/pgdg.list || echo 0 && apt update && apt
+                install -y curl
+      terminationMessagePath: /dev/termination-log
+      terminationMessagePolicy: File
+      imagePullPolicy: IfNotPresent
+  restartPolicy: Always
+  terminationGracePeriodSeconds: 30
+  dnsPolicy: ClusterFirst
+  serviceAccountName: default
+  serviceAccount: default
+  securityContext: {}
+  """
+    }
+  }
+
+  stages {
+    stage ('Build Enviroment') {
+      steps {
+        container('compiler') {
+          script {
+            sh './utils/repo-update-scripts/build-update-github.sh ${MODULE_PACKAGE} ${COMMIT_INPROGRESS_STATUS} "Starting build" $ACCESS_TOKEN $GIT_COMMIT $BUILD_URL ${CONTEXT_BUILD}'
```

Comment:
```suggestion
            sh './utils/repo-update-scripts/build-update-github.sh ${MODULE_PACKAGE} ${COMMIT_INPROGRESS_STATUS} "Starting build" $ACCESS_TOKEN $GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
```

Reply:
It's catching the env variable this way:
```
+ ./utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker success Successful Docker Test **** 2b21e0069478aa1f9af8d94de7836b2fb1deb17c http://jenkins2.etendo.cloud/job/etendo_docker_test/job/feature%252FEPL-1696/14/ Module Tests
{"state":"success", "target_url":"http://jenkins2.etendo.cloud/job/etendo_docker_test/job/feature%252FEPL-1696/14/", "description":"Successful Docker Test", "context":"Module Tests"}
```

I mean, without escaping the double quotes.